docs/fixed_point.rst

```diff
@@ -63,15 +65,19 @@ Anderson Acceleration
 
     jaxopt.AndersonAcceleration
 
-Anderson acceleration is an iterative method that aims to compute the next iterate :math:`x_{n}` as
-a linear combination of the :math:`m` last iterates :math:`[x_{n-m},x_{n-m+1},\ldots x_{n-1}]`. The optimal coefficients
-of the linear combination are computed 'on the fly' at each iteration. As a result, not only the convergence is faster
-but the convergence conditions are weakened, allowing to tackle problems PicardIterations could not.
-See `this paper <https://arxiv.org/abs/1909.04638>`_ for more details.
-  
-The size of the history :math:`m` (denoted ``history_size`` below) plays a crucial role in method performance. 
-An higher :math:`m` could speed up the convergence at the cost of higher memory consumption, and more numerical instabilities.
-Those numerical instabilities can be mitigated by increasing the ``ridge`` regularization hyper-parameter.
+Anderson acceleration is an iterative method that aims to compute the next
+iterate :math:`x_{n}` as a linear combination of the :math:`m` last iterates
+:math:`[x_{n-m},x_{n-m+1},\ldots x_{n-1}]`. The optimal coefficients of the
```

Comment:
I would remove optimal from here since we're not really explaining in which way we're optimal. I would say "The coefficients of the ..."